models/BaseEntity.cfc

```diff
@@ -761,15 +778,19 @@ component accessors="true" {
         param md.readonly = false;
         setReadOnly( md.readonly );
         param md.properties = [];
+
+
         setAttributesFromProperties( md.properties );
     }
 
+
     private function setAttributesFromProperties( properties ) {
-        return setAttributes(
+
+        setAttributes(
             properties.reduce( function( acc, prop ) {
                 param prop.column = prop.name;
                 param prop.persistent = true;
-                if ( prop.persistent ) {
+                if ( javacast( "boolean", prop.persistent ) ) {
```

Comment:
What cases are we covering with the `javacast`?

Reply:
String values in property anotations if they don't specify a `type` attribute:

```
property name="myProp" persistent="true";
```

There are some inconsistencies with different engines on that, but javacast fixes them all.